apps/ejabberd/src/mod_roster_mnesia.erl

```diff
@@ -56,14 +56,10 @@ read_roster_version(LUser, LServer) ->
         [] -> error
     end.
 
-write_roster_version(LUser, LServer, InTransaction, Ver) ->
-    US = {LUser, LServer},
-    if InTransaction ->
-           mnesia:write(#roster_version{us = US, version = Ver});
-       true ->
-           mnesia:dirty_write(#roster_version{us = US,
-                                              version = Ver})
-    end.
+write_roster_version(LUser, LServer, true, Ver) ->
```

Comment:
Or maybe we could replace the boolean for `IsTransaction` with some meaningful atom? Like `in_transaction` and `no_transaction`?